### PR TITLE
Makes russian work properly

### PR DESCRIPTION
**Pull request by SireTurret2**
РАЗ РАЗ РАЗ ЭТО ХАРДБАСС

### CODE REVIEW

**Comment by SireTurret2** [comment]
@mattroks101

**Comment by SireTurret2** [comment]
Screenshot_19
Screenshot_20

**Comment by mattroks101** [comment]
Does this break normal english?

File: code/_helpers/text.dm [hunk 1]
```diff
@@ -207,6 +207,15 @@
 		t = "[t] "
 	return t
 
+/proc/add_characters(c, n = 0) //Adds whatever character in 'c' and repeats that process 'n' times then returns text string as "XXXXXXXXXX"
```
Comment (mattmatterson111) [inline code/_helpers/text.dm]: I don't see this proc being called anywhere. Am I missing something?

File: code/_helpers/text.dm [hunk 2]
```diff
@@ -227,7 +236,7 @@
 
 //Returns a string with the first element of the string capitalized.
 /proc/capitalize(var/t as text)
-	return uppertext(copytext(t, 1, 2)) + copytext(t, 2)
+	return uppertext(copytext_char(t, 1, 2)) + copytext_char(t, 2)
```
Comment (mattmatterson111) [inline code/_helpers/text.dm:230]: This looks like the only thing that's actually doing anything, what's that other thing for?

File: code/_helpers/text.dm (hunk 1 above)
Comment (SireTurret2) [reply code/_helpers/text.dm]: Removed it. It was meant to be used for nuclear codes.

**Pull request merged**